gapic/api/artman_bigtable.yaml

```diff
@@ -1,6 +1,7 @@
 common:
   api_name: bigtable
   api_version: v2
+  package_type: grpc_client
```

Comment:
Right... Is it possible to use [common.yaml](https://github.com/googleapis/googleapis/blob/master/gapic/lang/common.yaml) to store default values? I think that would be a better option than implementing the logic in `artman`, since you'd have to change the config parsing code to avoid the runtime exception. WDYT?